cnames_active.js

```diff
@@ -2020,6 +2020,7 @@ var cnames_active = {
   "nested-error": "ultirequiem.github.io/nested_error",
   "neteasecloudmusicapi": "w4ctech.github.io/NeteaseCloudMusicApi",
   "neurax-api": "configneurax.netlify.app",
+  "neureact": "kasmirawijayathunga.github.io/neureact"
```

Comment:
```suggestion
  "neureact": "kasmirawijayathunga.github.io/neureact",
```
As the CI error indicates, you're missing a comma on this entry